share/cmake/patches/PatchMinizip-ngCMakelists.cmake

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: BSD-3-Clause
```

Comment:
The idea here is to change the CMake minimum version of MINIZIP-NG to 3.12.

EDIT:
But this is not working as it needs `target_link_directories` (one particular line in minizip-ng CMakelists.txt), and this is available from CMake 3.13+.

Reply:
Should we drop the VFX2019 Linux test and make room for upcoming VFX2023? Or just upgrade the VFX2019 docker image CMake version if possible?

Reply:
I've added this to the agenda of the Oct. 17 TSC meeting.